collections/_posts/2024-11-21-new-code-of-conduct-committee-members.md

```diff
@@ -0,0 +1,42 @@
+
+Since that time, the Typelevel steering committee voted in three new members to the Code of Conduct Committee! The committee now has five members total.
+
+Here are some brief introductions to our current committee members.
+
+#### Lucas Satabin (he/him)
+
+> I've been using Scala since 2009 both professionally and personally, and the Typelevel ecosystem since the scalac fork. Since then I have been trying to contribute libraries in the ecosystem on a regular basis. I currently work at [commercetools](https://commercetools.com/).
+>
+> I am based in the Paris area in France, and in my free time I like to play board games, TTRPGs, CRPGs, and I read a lot of comic books (mostly European and American). Baking bread, making mustard, and brewing slow coffees are my favorite kitchen based activities. I also tend to talk too much about trains.
+
+#### Kateu Herbert (he/him)
+
+> I’ve been programming with Scala for about 5 years now. I am a statistician by profession, currently working as a data analyst for [Enveritas](https://www.enveritas.org/) based in Kampala, Uganda. I use Scala for all my personal projects and in my free time, I share my knowledge through [blog posts](https://hkateu.github.io/kateuherbert.github.io/). Other hobbies include playing around with Linux distros, training my dog, and watching and reading about tech related content.
+
+#### Arman Bilge (he/him)
+
+> I'm Arman, based in Seattle, and I have been involved with Typelevel for almost four years now. I was first drawn to contribute by the welcoming and friendly people in our Discord! This community is near and dear to me and I am committed to fostering a safe and inclusive environment where we can work, learn and play together :)
+
+#### Sam Pilsworth (she/her)
+
+> Hi, I'm Sam! I'm part of the Typelevel steering committee and now also the code of conduct committee! I care deeply about the Typelevel community and creating a welcoming and supportive environment for everyone. I live in Canada, and when I'm not doing computer related activities I like to read books and play with my dog.
+
+#### Andrew Valencik (he/him)
+
+> Hi, I'm Andrew, I live in Ottawa, Canada. I hang out around Typelevel because the people are kind and welcoming. I want to help keep our community a safe and fun space. I think the libraries are pretty cool too and enjoy contributing to open source when I can. When I'm not thinking about computers I'm probably thinking about food or my two cats.
+
+#### What does this mean for our community?
+Code of Conduct members will work in accordance with the new [Code of Conduct](https://github.com/typelevel/governance/blob/main/CODE-OF-CONDUCT.md) and [Enforcement Policy](https://github.com/typelevel/governance/blob/main/ENFORCEMENT-POLICY.md). 
```

Comment:
The diff doesn't render great for me, but I'm adding a newline below the header and ditching the word "new" here:

```suggestion

Code of Conduct members will work in accordance with the [Code of Conduct](https://github.com/typelevel/governance/blob/main/CODE-OF-CONDUCT.md) and [Enforcement Policy](https://github.com/typelevel/governance/blob/main/ENFORCEMENT-POLICY.md). 
```